.github/workflows/fluentd-standalone-image.yml

```diff
@@ -0,0 +1,45 @@
+
+name: Fluentd-Image
```

Comment:
I suggest to change the name to fluentd-standalone so we wont get confused since the daemonset image workflow has the same name